docs/content/en/docs/workflows/debug.md

```diff
@@ -343,6 +220,175 @@ your base image.  (`//` comments must be stripped.)
 ```
 {{< /alert >}}
 
+{{% /tab %}}
+{{% /tabs %}}
+
+## Supported Language Runtimes
+
+This section describes how `debug` recognizes the language runtime used in a
+container image for specific language runtimes.
```

Comment:
Rather than splitting this section out, why don't we integrate it into the previous tabs.  Users of most of the runtimes will have to look into this sections if only to ensure they set the right environment variables.

Reply:
I wanted to separate this out because this section is independently relevant to Cloud Code. So when debug isn't working in Cloud Code for a language runtime they can link this section directly. Having it merged with the previous section would present additional info that IDE users don't necessarily need to know immediately.